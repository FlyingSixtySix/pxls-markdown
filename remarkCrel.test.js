const { processor, expectProcessorOutput } = require('./testUtil');
const remarkCrel = require('./remarkCrel');

processor.use(remarkCrel);

test('compiles "test" correctly', () => expectProcessorOutput('test').toMatchSnapshot());
test('compiles "test" with ignoreParagraphs = true correctly', () => {
	const configuredProcessor = processor().use(remarkCrel, { ignoreParagraphs: true });
	expectProcessorOutput('test', configuredProcessor).toMatchSnapshot();
});
test('compiles "*test*" correctly', () => expectProcessorOutput('*test*').toMatchSnapshot());
test('compiles "__test__" with the underline plugin correctly', () => {
	const configuredProcessor = processor().use(require('./plugins/underline'));
	expectProcessorOutput('__test__', configuredProcessor).toMatchSnapshot();
});
