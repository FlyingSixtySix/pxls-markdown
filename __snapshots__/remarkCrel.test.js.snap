// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles "*test*" correctly 1`] = `
Array [
  <p>
    <i>
      test
    </i>
  </p>,
]
`;

exports[`compiles "__test__" with the underline plugin correctly 1`] = `
Array [
  <p>
    <u>
      test
    </u>
  </p>,
]
`;

exports[`compiles "test" correctly 1`] = `
Array [
  <p>
    test
  </p>,
]
`;

exports[`compiles "test" with ignoreParagraphs = true correctly 1`] = `
Array [
  "test",
]
`;
